core/tests/protractor_utils/workflow.js

```diff
@@ -57,13 +58,16 @@ var checkForAddTitleWarning = async function() {
 
 // Trigger onblur event for title.
 var triggerTitleOnBlurEvent = async function() {
-  await element(by.css('.protractor-test-exploration-title-input')).click();
-  await element(by.css('.protractor-test-exploration-objective-input')).click();
+  var tstExpTitI = element(by.css('.protractor-test-exploration-title-input'));
+  await action.click('Test Exploration Title Input', tstExpTitI);
+  var tstExOI = element(by.css('.protractor-test-exploration-objective-input'));
+  await action.click('Test Exploration Objective Input', tstExOI);
 };
 
 // Open edit roles.
 var openEditRolesForm = async function() {
-  await element(by.css('.protractor-test-edit-roles')).click();
+  var testEditRoles = element(by.css('.protractor-test-edit-roles'));
+  await action.click('Test Edit Roles', testEditRoles);
   await element(by.css('.protractor-test-role-username')).sendKeys(
     'Chuck Norris');
```

Comment:
Please use `action.sendKeys` here

Reply:
Can you share any guide about these which will be more helpful?

Reply:
Check the comment at the top of #12148 

![Screen Shot 2021-04-09 at 23 40 35](https://user-images.githubusercontent.com/19878639/114257294-051a9a00-998d-11eb-9524-4eb676957236.png)